storm-core/src/jvm/org/apache/storm/container/ResourceIsolationInterface.java

```diff
@@ -56,4 +64,13 @@
      */
     List<String> getLaunchCommandPrefix(String workerId);
 
+    /**
+     * Get the list of PIDs currently in an isolated container
+     * @param workerId the id of the worker to get these for
+     * @return the set of PIDs, this will be combined with
+     * other ways of getting PIDs. An Empty set or null if
+     * no PIDs are found.
+     * @throws IOException on any error
+     */
```

Comment:
I agree with @d2r, it's just as easy for the implementation to return Collections.emptySet() as null, and null checking can be easy to forget when the code is modified later.